cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -923,6 +876,9 @@ EOS
       field "mem_average" do
         label "Memory Average %"
       end
+      field "mem_p99" do
+        label "Memory p99"
+      end
       field "tags" do
         label "Tags"
       end
```

Comment:
Line 900 and 907 - escalations should have 'esc_' prefix. Seems it was missed on previous changes
Also remember to make the changes to Line 820 and 821 where they are referenced